app/domain/authentication/authn_oidc/get_user_details.rb

```diff
@@ -45,7 +45,8 @@ def user_details
           id_token: id_token,
```

Comment:
Rename UserDetails

Reply:
OidcToken changed to OidcConjurToken

UserDetails changed to OidcIDTokenDetails

Reply:
done